Sources/PenguinParallel/NonblockingThreadPool/NonBlockingThreadPool.swift

```diff
@@ -64,7 +64,13 @@ public class NonBlockingThreadPool<Environment: ConcurrencyPlatform>: ComputeThr
   let totalThreadCount: Int
   let externalFastPathThreadCount: Int
   var externalFastPathThreadSeenCount: Int = 0
-  let coprimes: [Int]
+
+  /// An array of step-sizes that are each coprime with `queues.count`.
```

Comment:
```suggestion
  /// A vocabulary of step sizes used to randomly search the list of queues for work to steal.
```

Talk about what *role* it plays, not the properties of its value.  You may need to adjust the explanation below to introduce coprimality if you accept this edit.